### DISCUSSION TITLE
SPARQL over HDT: Are named source select query supported ?

### DISCUSSION THREAD

**Discussion by davidandreoletti** [the-qa-company/qEndpoint#139] [Q&A] [answered]
Hi,

Currently using qEndpoint to run SPARQL queries (SELECT ...) **over HDT file backed store.**

1. Is ```FROM NAMED``` supported ?2. 

```
SELECT ...
    FROM <...>
    FROM NAMED <...>
    WHERE {
...
}
```

1.1. How is the mapping between the HDT file and named source performed ?

**Comment by D063520** [comment, marked as answer]
Hi,

we do not support NAMED Graphs. Trying to insert and query named graphs might give unpredictable results.

Salut
D063520
